src/env-inl.h

```diff
@@ -174,6 +174,13 @@ inline bool TickInfo::has_rejection_to_warn() const {
   return fields_[kHasRejectionToWarn] == 1;
 }
 
+inline Environment* Environment::GetCurrent(v8::Local<v8::Value> value) {
```

Comment:
Is the expectation that an v8::Local<v8::Value> instance allows us to retrieve an associated v8::Context?

Reply:
@lemire Take a look into: https://github.com/nodejs/node/compare/main...anonrig:node:yagiz/try-environment?expand=1

I'm trying to store the environment pointer as a v8::External in FunctionTemplate (just as @devsnek recommended) so that we can bypass HandleScope creation for getting Environment* in fast api calls.

v8::External extends from v8::Value. So, we can technically convert v8::Local<v8::External> (which is Environment*) to Environment*